ivy-core/src/main/java/com/ivy/core/data/model/NumpadType.kt

```diff
@@ -0,0 +1,20 @@
+package com.ivy.core.data.model
+
+enum class NumpadType(val layout: Array<String>) {
```

Comment:
Use `ImmutableList` otherwise Compose will consider it unstable and recompose like crazy 😬 
Also explicitly mark it with `@Immutable`